us/en/skymiles/scripts/scripts.js

```diff
@@ -84,14 +84,14 @@ function decorateInlineToggles(container) {
     .forEach(createInlineToggle);
 }
 
-function decorateScreenReaderOnly(container) {
-  [...container.querySelectorAll('del')].forEach((el) => {
-    const span = document.createElement('span');
-    span.classList.add('sr-only');
-    span.innerHTML = el.innerHTML;
-    el.replaceWith(span);
-  });
-}
+// function decorateScreenReaderOnly(container) {
```

Comment:
I fixed the linting in main for this with an eslint disable. So you can probably just ignore this and revert the changes or rebase/merge the branch

Reply:
yes, I rebased my branch, fixed in [f1d8070](https://github.com/hlxsites/delta/pull/89/commits/f1d80700d4d0f621c6466bffc47d9c5b6e4f461c)